docker/mysql/mysql.cnf

```diff
@@ -13,3 +13,4 @@ collation-server = utf8mb4_unicode_ci
 
 character-set-client-handshake = FALSE
 init_connect = "SET NAMES utf8mb4 COLLATE utf8mb4_unicode_ci"
+sql_mode = IGNORE_SPACE,STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION
```

Comment:
Before merging:
* We should verify whether or not this is still needed with MariaDB.
* We should ensure this PR works with MariaDB, as there were reported discrepancies between MySQL and MariaDB during testing.

Reply:
sql mode is set on the connection level in the database config - server sql mode should stay as-is.
I'll look into that further

Reply:
Confirmed this line of config no longer seems necessary. Removed in https://github.com/RetroAchievements/RAWeb/pull/1862/commits/63c22cc0e23e87ad97eecacae8336b6963111cc6.